pandas/core/indexes/multi.py

```diff
@@ -1412,6 +1412,45 @@ def from_product(cls, iterables, sortorder=None, names=None):
         labels = cartesian_product(labels)
         return MultiIndex(levels, labels, sortorder=sortorder, names=names)
 
+    @classmethod
+    def from_frame(cls, df, squeeze=True):
+        """
+        Make a MultiIndex from a dataframe
+
+        Parameters
+        ----------
+        df : pd.DataFrame
+            DataFrame to be converted to MultiIndex
+        squeeze : bool
+            If df is a single column, squeeze multiindex to be a regular index.
+
+        Returns
+        -------
+        index : MultiIndex
+
+        Examples
+        --------
+        >>> df = pd.DataFrame([[0, u'green'], [0, u'purple'], [1, u'green'],
+                               [1, u'purple'], [2, u'green'], [2, u'purple']],
+                              columns=[u'number', u'color'])
```

Comment:
this is missing the `...` for multine commands. Also  show the content of `df` after creating it.